src/page-merger.js

```diff
@@ -0,0 +1,62 @@
+class PageMerger {
+  constructor(paginator) {
+    this._count = null;
+    this._paginator = paginator;
+  }
+
+  merge(startPage, endPage) {
+    return this._fetchRequests(startPage, endPage)
+      .then(this._makeCollection.bind(this))
```

Comment:
This is just me sticking to the no anon function rule too strictly. I'm glad we're changing that.